cdserver/features/authentication/login.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Login
+  As a user,
+  I want to log in with my email and password
+
+Scenario: successful user login
+  Given The following user exists in the system
+    | email           | password |
+    | joeblow@hbo.com | secret   |
+  When I login as the following user
+    | email           | password |
+    | joeblow@hbo.com | secret   |
+  Then I should receive the following response
+    | status | 201            |
```

Comment:
This should be 201 Created
Can you make the test lookup the human readable description of the status?  I believe there is a simple ruby API for that.  Might be an enum